validator/storage/share.go

```diff
@@ -25,18 +25,20 @@ func (keys PubKeys) Aggregate() bls.PublicKey {
 
 // Share storage model
 type Share struct {
-	NodeID    uint64
-	PublicKey *bls.PublicKey
-	Committee map[uint64]*proto.Node
-	Metadata  *beacon.ValidatorMetadata // pointer in order to support nil
+	NodeID       uint64
+	PublicKey    *bls.PublicKey
+	Committee    map[uint64]*proto.Node
+	Metadata     *beacon.ValidatorMetadata // pointer in order to support nil
+	OwnerAddress string
```

Comment:
what will happened for old struct that saved in storage without the owner addr?

Reply:
we are going to run the node with flag `clear registry data`  by default enabled